addons/base_import_module/tests/test_import_module.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+
+import odoo.tests
+from odoo.tools import file_open
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestImportModule(odoo.tests.TransactionCase):
+
+    def test_import_and_uninstall_module(self):
+        IrModule = self.env['ir.module.module']
+        module = 'test_module'
+        bundle = 'web.assets_backend'
+        path = '/test_module/static/src/js/test.js'
+        file_content = b"console.log('test_module javascript');\n"
+
+        # Import test module
+        with file_open('base_import_module/tests/test_module.zip', 'rb') as zip_file:
+            IrModule.import_zipfile(zip_file)
+
+        attachment = self.env['ir.attachment'].search([('url', '=', path)])
+        self.assertEqual(len(attachment), 1)
+        self.assertEqual(attachment.name, 'test.js')
+        self.assertEqual(attachment.type, 'binary')
+        self.assertEqual(base64.decodebytes(attachment.datas), file_content)
```

Comment:
1. avoid `base64.decodebytes`, prefer` b64decode` for clarity, and validate the decoding (that is not done by default)
2. this isn't even necessary, `attachment,raw` should provide the decoded content